CHANGELOG.md

```diff
@@ -2,18 +2,11 @@
 
 ##  0.8.0 (2018-06-07)
```

Comment:
It actually went out a few days later.

Reply:
you are right... we use an automated tool to generate the changelog, so not sure why this would be out of date. Thanks for pointing this out.